packages/xo-web/src/xo-app/backup/new/mirror/index.js

```diff
@@ -199,16 +225,55 @@ const NewMirrorBackup = decorate([
       },
       resetMirrorBackup: () => (_, props) => getInitialState(props),
       linkState,
-      toggleMode: (_, { mode }) => ({ mode }),
+      toggleMode: (_, { mode }) => ({ mode, vmsToMirror: undefined }),
+      toggleMirrorAll: (_, value) => ({ mirrorAll: value, vmsToMirror: undefined }),
```

Comment:
Avoid clearing `vmsToMirror` when changing `mirrorAll`, I don't think this is really necessary and it will annoy users if they misclick on it.